tests/web_resources/test_json_resource.py

```diff
@@ -17,7 +17,7 @@ def unnest_data(self, data, destination):
         pass
 
     def __init__(self):
-        super().__init__(self.name, self.base_url, self.endpoint, self.version)
+        super().__init__(self.name, self.base_url, self.endpoint, self.version, self.requested_at)
```

Comment:
hmm something seems funky here, given that `JSONResource.__init__` does not take a `requested_at` parameter.
https://github.com/phillymedia/vestapol/blob/quickstart/vestapol/web_resources/json_resource.py#L13-L15